apps/prairielearn/assets/scripts/instructorFileEditorClient.ts

```diff
@@ -69,11 +71,11 @@ class InstructorFileEditor {
       autoScrollEditorIntoView: true,
       wrap: true,
       showPrintMargin: false,
-      mode: aceMode || undefined,
+      mode: aceMode ?? 'ace/mode/text',
       readOnly,
       enableKeyboardAccessibility: true,
       theme: 'ace/theme/chrome',
-    });
+    } satisfies Partial<ace.Ace.EditorOptions>);
```

Comment:
This gives us better type-checking; `ace.edit` has its second argument typed as `any`.